src/main/java/it/pagopa/transactions/services/TransactionsService.java

```diff
@@ -137,10 +141,18 @@ public Mono<TransactionInfoDto> getTransactionInfo(String transactionId) {
                                                         .amount(noticeCode.getAmount())
                                                         .reason(noticeCode.getDescription())
                                                         .paymentToken(noticeCode.getPaymentToken())
-                                                        .authToken(null)
                                                         .rptId(noticeCode.getRptId())
                                         ).toList()
                                 )
+                                .amountTotal(transaction.getAmountTotal())
+                                .feeTotal(transaction.getFeeTotal())
+                                .origin(
+                                        TransactionInfoDto.OriginEnum.valueOf(
+                                                Optional.ofNullable(transaction.getOrigin()).orElse(
+                                                        it.pagopa.ecommerce.commons.documents.Transaction.OriginType.UNKNOWN
+                                                ).toString()
```

Comment:
Isn't `valueOf` redundant?
```suggestion
Optional.ofNullable(transaction.getOrigin()).orElse(it.pagopa.ecommerce.commons.documents.Transaction.OriginType.UNKNOWN).toString()
```

Reply:
Moreover: I wouldn't even bother checking wether `origin` is null, given that we always populate this field when initializing a transaction its absence would be a bug, so in that case a `NullPointerException` would be fine

Reply:
No because the original value `it.pagopa.ecommerce.commons.documents.Transaction.OriginType` is from documents package of commons. Here I specified in the yaml file the same enum, but it comes from another package. 
So I had 2 chances:
1) take the same enum from commons package. This indeed could be fine, but I need help to include it in the yaml file
2) do it as it is now.

Reply:
You're right, my bad :)
1. I don't know if it's even possible, 2. is fine, but what do you think about this:

```suggestion
TransactionInfoDto.OriginEnum.valueOf(transaction.getOrigin()).toString()
```

The thing I'm worried about is by using `Optional` we would mask a potential bug

Reply:
I did it in this way 👍